viam-orb-slam3_test.go

```diff
@@ -178,17 +153,6 @@ func setupDeps(attr *slamConfig.AttrConfig) registry.Dependencies {
 				return camera.Properties{}, nil
```

Comment:
`good_lidar` is kept because of the `New orbslamv3 service with lidar without Next implementation` test. I had originally removed this test and `good_lidar` however Kat mentioned that it was a good idea to keep these. https://github.com/viamrobotics/viam-orb-slam3/pull/19#discussion_r1137603062

Reply:
[nit] I believe, though, that all that's necessary to keep from `good_lidar` is the `ProjectorFunc` (correct me if I'm wrong). What I'm trying to say is: Only overwrite the functions that you need for your test.